content/en/docs/Tasks/adding-an-admission-webhook.md

```diff
@@ -0,0 +1,83 @@
+# Admission Webhooks
+
+## Background
+
+OLM is capable of managing the lifecycle of [validating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) and [mutating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) admission webhooks that are shipped alongside your operator.
+
+You may be unfamiliar with webhooks. At a high level, after a request has been authenticated and authorized, admission webhooks intercept requests against the Kubernetes API and have an opportunity to validate or update the object before it is saved in the object store. Please refer to the following table that highlights what each webhook is capable of:
```

Comment:
Not sure if we need the sentence `You may be unfamiliar with webhooks`. 

Also, seems like a majority of the `Background` should be moved to the References section of the doc site, and doc can link to the Reference.